### DISCUSSION TITLE
Export/Import custom effects between worlds?

### DISCUSSION THREAD

**Discussion by ShannonMcG** [DFreds/dfreds-convenient-effects#127] [Q&A] [answered]
First, thanks for the awesome module. 

I've made a few custom effects which work great. But now I want to also use them in my other worlds.

Is there a way to export/import custom effects between worlds?

**Comment by DFreds** [comment, marked as answer]
You can expand the Custom folder in the Convenient Effects app and press the export button located in the expanded folder. Then load up your new world and press the import button

**Reply by ShannonMcG** [reply]
Oh snapples, I totally overlooked those buttons. You DO think of everything. Thanks!
